src/app/components/list-items/list-items.component.ts

```diff
@@ -1,61 +1,86 @@
-import { DialogMaterialComponent } from "./../dialog-material/dialog-material.component";
-import { Component, OnInit } from "@angular/core";
-import { CdkDragDrop, moveItemInArray } from "@angular/cdk/drag-drop";
-import {
-  MatDialog,
-  MatDialogRef,
-  MAT_DIALOG_DATA,
-} from "@angular/material/dialog";
-import { FormBuilder, FormGroup, Validators } from "@angular/forms";
 
+import { Observable } from 'rxjs';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+//Angular Material
+import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
+import { DialogMaterialComponent } from './../dialog-material/dialog-material.component';
+import { MatDialog} from '@angular/material/dialog';
+//Ngrx
+import { Store } from '@ngrx/store';
+import { IAppState } from 'src/app/store/state/app.state';
+import { RouterGo } from 'ngrx-router';
+import { selectList } from './../../store/selectors/list.selectors';
```

Comment:
(Opcional) Todo good. Una sugerencia es mejorar los imports abreviándolos. Te dejo este artículo :) https://netbasal.com/sexier-imports-in-typescript-e3c645bdd3c6